src/precice/impl/ParticipantImpl.cpp

```diff
@@ -1553,6 +1568,7 @@ void ParticipantImpl::syncTimestep(double computedTimeStepSize)
 void ParticipantImpl::advanceCouplingScheme()
 {
   PRECICE_DEBUG("Advance coupling scheme");
+  Event e("advanceCoupling", profiling::Fundamental);
```

Comment:
This can contain multiple coupling schemes and is hence more of an informational "what is going on"-event and less useful for measurements.